### ISSUE TITLE
dotnet publish won't work with this package

### ISSUE THREAD

**Issue by msitsoftware** [Morasiu/BlazorTimeline#7]
I have a problem using this package with dotnet publish. I got the error C:\Program Files\dotnet" exited with code 1. That's it. When I remove the package everything is working fine.

**Comment by Morasiu** [comment]
Hello.

Do you have a specific error? Also can you provide whole command which you are using to publish?

I just tried it in my Demo project using:

```bash
dotnet publish Demo\Demo.csproj
```

and... it works. 

My .Net Version: 5.0.104

**Comment by msitsoftware** [comment]
No I only got the error I already mentioned. I'm using .NET 5.0.302.

The command I use is:
```
dotnet publish myProject.csproj
```

**Comment by Morasiu** [comment]
Seems to be a problem with dotnet. It works in version 5.0.104 but in 5.0.302 (using `dotnet publish -v d`) it returns:

```
 Could not find file 'C:\Users\Morasiu\.nuget\packages\blazortimeline\1.3.1\staticwebassets\.gitkeep'.
            at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
            at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
            at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
            at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
            at System.IO.File.OpenRead(String path)
            at Microsoft.NET.Sdk.BlazorWebAssembly.Tools.BrotliCompressCommand.<ExecuteCoreAsync>b__16_0(Int32 i)
            at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`1.<ForWorker>b__1(RangeWorker& currentWorker, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
         --- End of stack trace from previous location ---
            at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`1.<ForWorker>b__1(RangeWorker& currentWorker, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
            at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean& yieldedBeforeCompletion)
            at System.Threading.Tasks.TaskReplicator.Replica.Execute()

```

I guess I should report this bug.

**Comment by Morasiu** [comment]
Reported: https://github.com/dotnet/sdk/issues/19453

**Comment by Morasiu** [comment]
Fixed in version 1.3.2
